### PR TITLE
Resolve the URL issue in the WSClientWinStore_sample and ProjectX_sample.

### PR DESCRIPTION
**Pull request by SandeepNayak077**
Implemented following code to resolve the URL issue in the WSClientWinStore_sample.
- Resolved URL issue in the *.master pages of the application
- Implemented baseUrl property to resolve the URL issue in the javascript and *.html pages
- Implemented BaseUrl property to resolve the URL issue of the Service References.

Added baseUrl property in the rootMasterPage.master of ProjectX_sample to resolve the URL issue of progress loading.

### CODE REVIEW

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Please check.
https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/107/files

**Comment by yusukemaegawa** [comment]
@daisukenishino 
@SandeepNayak077 

You implemented codes in root_VS2013 only.
Don't you implement code in root_VS2012 and in root_VS2015?

**Comment by daisukenishino** [comment]
@yusukemaegawa 

After completing the VS2013 correspondence of [this issue](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/12),
We will schedule to be reflected to the VS2010 and Store app of VS2012.
Later, this will be reflected to the entire of repository by ride the [merge route](https://github.com/OpenTouryoProject/OpenTouryo/issues/131).

File: root_VS2013/programs/C#/Samples/WS_sample/WSClient_sample/WSClientWinStore_samples/WSClientWinStore_sample/Service References/SbServiceReference/Reference.cs [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ public partial class ServiceForSbSoapClient : System.ServiceModel.ClientBase<WSC
         
         private static System.ServiceModel.EndpointAddress GetEndpointAddress(EndpointConfiguration endpointConfiguration) {
             if ((endpointConfiguration == EndpointConfiguration.ServiceForSbSoap)) {
-                return new System.ServiceModel.EndpointAddress("http://localhost:9996/WSClientSL_sample/WebService/ServiceForSb.asmx");
+                return new System.ServiceModel.EndpointAddress(App.BaseUrl + "WebService/ServiceForSb.asmx");
```
Comment (yusukemaegawa) [inline root_VS2013/programs/C#/Samples/WS_sample/WSClient_sample/WSClientWinStore_samples/WSClientWinStore_sample/Service References/SbServiceReference/Reference.cs]: @SandeepNayak077 

When developer adds the service reference to web service, Visual Studio generates this file automatically.
So, if developer updates the service reference, Visual Studio regenerates this file and overwrites this settings.

Should developer modify this file again as follows?
```C#
return new System.ServiceModel.EndpointAddress(App.BaseUrl + "[new url to web service]");
```

Comment (yusukemaegawa) [inline root_VS2013/programs/C#/Samples/WS_sample/WSClient_sample/WSClientWinStore_samples/WSClientWinStore_sample/Service References/SbServiceReference/Reference.cs]: @SandeepNayak077 

In [OpenTouryoProject/OpenTouryoTemplates/issues/12](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/12), @daisukenishino describes as follows.

>- Service Reference of correspondence is not required.
>   - SOPA Web API of [Silverlight](https://github.com/OpenTouryoProject/OpenTouryoTemplates/tree/master/root_VS2013/programs/C%23/Samples/WS_sample/WSClient_sample/WSClientSL_samples/WSClientSL_sample/) and [Windows Store app](https://github.com/OpenTouryoProject/OpenTouryoTemplates/blob/master/root_VS2013/programs/C%23/Samples/WS_sample/WSClient_sample/WSClientWinStore_samples/WSClientWinStore_sample/) is used this class.

Do you need to modify this file?

**Comment by SandeepNayak077** [comment]
@daisukenishino 
@yusukemaegawa 

>https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/107#discussion_r55156764
>https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/107#discussion_r55157672

Yes that's correct Visual Studio regenerates and overwrites the service reference file.
And the developer must modify the service reference file.
```C#
return new System.ServiceModel.EndpointAddress(App.BaseUrl + "[new url to web service]");
```

But, this change is not required, as per the @daisukenishino described in the  [OpenTouryoProject/OpenTouryoTemplates/issues/12](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/12). 
Thank you for your valuable information.
Sorry for the inconvenience, we will modify and send this correspondence in the next pull request.
Please confirm and merge this pull request asap.

**Comment by daisukenishino** [comment]
@SandeepNayak077 

Certainly, I described as follows.
- https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/12#issuecomment-183156143

(I am a bit clarify this.)

But I have merged this so this modification is good.
Therefor, revert is not required.

**Pull request merged**